azure-pipelines.yml

```diff
@@ -0,0 +1,50 @@
+jobs:
+  - job: Build
+    displayName: 'Build'
+    pool: Cloud-Build
+    strategy:
+      matrix:
+        Release-Any:
+          Platform: 'Any CPU'
+          BuildConfiguration: 'Release'
+    workspace:
+      clean: outputs
+
+    steps:
```

Comment:
Can we add git cleaning to this pipeline?

Reply:
Solved in: a7a51db85969d53eab02299daa68d6355cb5f3e3